pennylane/measurements/classical_shadow.py

```diff
@@ -219,50 +218,90 @@ class ShadowMeasurementProcess(MeasurementProcess):
     Args:
         args (tuple[Any]): Positional arguments passed to :class:`~.pennylane.measurements.MeasurementProcess`
         seed (Union[int, None]): The seed used to generate the random measurements
-        H (:class:`~.pennylane.Hamiltonian` or :class:`~.pennylane.operation.Tensor`): Observable
-            to compute the expectation value over. Only used when ``return_type`` is ``ShadowExpval``.
-        k (int): Number of equal parts to split the shadow's measurements to compute the median of means.
-            ``k=1`` corresponds to simply taking the mean over all measurements. Only used
-            when ``return_type`` is ``ShadowExpval``.
         kwargs (dict[Any, Any]): Additional keyword arguments passed to :class:`~.pennylane.measurements.MeasurementProcess`
     """
 
-    def __init__(self, *args, seed=None, H=None, k=1, **kwargs):
+    def __init__(self, *args, seed=None, **kwargs):
         self.seed = seed
-        self.H = H
-        self.k = k
         super().__init__(*args, **kwargs)
 
-    @property
-    def numeric_type(self):
-        """The Python numeric type of the measurement result.
-
-        Returns:
-            type: This is ``int`` when the return type is ``Shadow``,
-            and ``float`` when the return type is ``ShadowExpval``.
+    def process(self, tape, device):
         """
-        return int if self.return_type is Shadow else float
+        Returns the measured bits and recipes in the classical shadow protocol.
 
-    def shape(self, device=None):
-        """The expected output shape of the ShadowMeasurementProcess.
+        The protocol is described in detail in the `classical shadows paper <https://arxiv.org/abs/2002.08953>`_.
+        This measurement process returns the randomized Pauli measurements (the ``recipes``)
+        that are performed for each qubit and snapshot as an integer:
+
+        - 0 for Pauli X,
+        - 1 for Pauli Y, and
+        - 2 for Pauli Z.
+
+        It also returns the measurement results (the ``bits``); 0 if the 1 eigenvalue
+        is sampled, and 1 if the -1 eigenvalue is sampled.
+
+        The device shots are used to specify the number of snapshots. If ``T`` is the number
+        of shots and ``n`` is the number of qubits, then both the measured bits and the
+        Pauli measurements have shape ``(T, n)``.
+
+        This implementation leverages vectorization and offers a significant speed-up over
+        the generic implementation.
+
+        .. Note::
+
+            This method internally calls ``np.einsum`` which supports at most 52 indices,
+            thus the classical shadow measurement for this device supports at most 52
+            qubits.
```

Comment:
This looks like it's copied from the `default.qubit` docstring instead of the generic `_qubit_device` docstring.

Reply:
Fixed!